Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -79,7 +79,7 @@ public abstract class BaseControllerPointer : ControllerPoseSynchronizer, IMixed
         /// </summary>
         /// <remarks>This <see href="https://docs.unity3d.com/ScriptReference/GameObject.html">GameObject</see> must have a <see cref="Microsoft.MixedReality.Toolkit.Input.IMixedRealityCursor"/> attached to it.</remarks>
         /// <param name="newCursor">The new cursor</param>
-        public virtual void SetCursor(GameObject newCursor = null)
+        private void SetCursor(GameObject newCursor = null)
```

Comment:
Hey, thanks for making a lot of these changes. I noticed that some formerly public functions are still private. Can you revert these changes, since they are technically breaking API changes?

This is just following up on Kurtis's observations from before.

Reply:
Alright, thanks for the review!